delta/plugins/search/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/search/model/SearchConfig.scala

```diff
@@ -38,6 +58,7 @@ object SearchConfig {
       query         <- loadSparqlQuery(pluginConfig.getString("indexing.query"))
       context       <- loadOption(pluginConfig, "indexing.context", loadExternalConfig[JsonObject])
       defaults      <- loadDefaults(pluginConfig)
+      suites        <- IO.fromEither(suiteSource.load[Suites]).mapError(InvalidSuites)
```

Comment:
the `loadXXX` methods are making this very readable, it's probably better to match that style